main.c

```diff
@@ -47,6 +47,7 @@ usage(void)
 	    "usage: scrypt {enc | dec} [-f] [-M maxmem]"
 	    " [-m maxmemfrac]\n"
 	    "              [-t maxtime] [-v] [-P] infile [outfile]\n"
+	    "       scrypt dec --print-params-only infile\n"
```

Comment:
I'd be more inclined to make this `scrypt info infile`; it seems odd to say "decrypt, except don't actually decrypt" when we just mean "print information about this encrypted file".

Reply:
done